CHANGELOG.md

```diff
@@ -52,10 +53,7 @@ The types of changes are:
 * Added a page on implementing and opting out of fideslog analytics [#751](https://github.com/ethyca/fidesops/pull/751)
 
 ### Fixed
-* Make reading of environment variables case insensitive [#712](https://github.com/ethyca/fidesops/pull/712)
-* Fix console warning in disable connection modal [#750](https://github.com/ethyca/fidesops/pull/750)
-* Fix no such container error with docker-compose [#758](https://github.com/ethyca/fidesops/pull/758)
-* Fixed issue with extending the configuration [#721](https://github.com/ethyca/fidesops/pull/721)
```

Comment:
@pattisdr it looks like these lines were removed from the changelog, is this intentional?

Reply:
good catch thank you, just a really messy merge